SI/Sem.02/Pract.02/task_10_p02.cpp

```diff
@@ -0,0 +1,29 @@
+
+
+#include <iostream>
+
+int main()
+{
+    int a = 0;
+    int b = 0;
+    std::cout << "a = ";
+    std::cin >> a;
```

Comment:
The point of the task is to receive from the console a, b and character that is +,-,*,/ and based on the characted you should do a specific operation